e2e/__tests__/callDoneTwice.test.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import wrap from 'jest-snapshot-serializer-raw';
+import runJest from '../runJest';
+import {extractSummary} from '../Utils';
+
+test('`done()` should not be called more than once', () => {
+  const {exitCode, stderr} = runJest('call-done-twice', [
+    `--testRunner='jest-circus/runner'`,
+  ]);
```

Comment:
```suggestion
  const {exitCode, stderr} = runJest('call-done-twice');
```

just call `skipSuiteOnJasmine();` above the test declaration